web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -735,7 +742,7 @@ const TeamReviews = ({ onBack, periodId }) => {
             key={reviewer.id}
             label={reviewer.name}
             variant="outlined"
-            onDelete={() => deleteReviewer(member, reviewer)}
+            onDelete={canUpdate ? () => deleteReviewer(member, reviewer) : null}
```

Comment:
Lots of UI elements are not either hidden or disabled based on the values of `canUpdate` and `canLaunch`.